index.js

```diff
@@ -7,21 +7,69 @@ const data = require('./data');
 
 const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
 
+const newRecipe = {
+  title: "Spanish Omelet",
+  level: "Amateur Chef",
+  cuisine: "Spanish",
+  dishType: "main_course",
+  image: "https://images.media-allrecipes.com/userphotos/720x405/815964.jpg",
+  duration: 30,
+}
+
+
 // Connection to the database "recipe-app"
 mongoose
   .connect(MONGODB_URI, {
     useCreateIndex: true,
     useNewUrlParser: true,
-    useUnifiedTopology: true
+    useUnifiedTopology: true,
+    useFindAndModify: false
   })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
+  
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    Recipe
+      .create(newRecipe)
+      .then(newRecipe => {
+        console.log(`The new recipe is: ${newRecipe.title}`)
+        return Recipe.insertMany(data)
+      })
+    
+      .then(newRecipes => {
+        newRecipes.forEach(
+          recipe => console.log(recipe.title)
+        )
+        return Recipe.findOneAndUpdate({ title: "Rigatoni alla Genovese" }, { duration: 100 }, {new:true})
+      })
+        
+      .then(updatedRecipe => {
+        console.log(`Success! ${updatedRecipe.title} was updated, it's duration is now ${updatedRecipe.duration}`)
+        return Recipe.deleteOne({ title: "Carrot Cake" })
+      })
+        
```

Comment:
```suggestion
      .then((updatedRecipe) => {
        console.log(
          `Success! ${updatedRecipe.title} was updated, it's duration is now ${updatedRecipe.duration}`
        );
      })
      .then(() =>  Recipe.deleteOne({ title: "Carrot Cake" }))
```
El` .deleteOne` podrías ubicarlo en otra promesa.